src/TesterInternal/App.config

```diff
@@ -1,11 +1,17 @@
-<?xml version="1.0"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
   </startup>
   <runtime>
-    <ThrowUnobservedTaskExceptions enabled="false"/>
-    <gcServer enabled="true"/>
-    <gcConcurrent enabled="false"/>
+    <ThrowUnobservedTaskExceptions enabled="false" />
+    <gcServer enabled="true" />
+    <gcConcurrent enabled="false" />
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
```

Comment:
We use one single version 5.0.8 of Newtonsoft.Json package in the whole code base.

The reason we do that is precisely to avoid needing to add wonky binding redirects everywhere.

Is there a reason why the standard v5.0.8 package can't be used here?

If we do want / need to use a different version of JSON library, then we should change everywhere to consistently use the new version, not just one single place.